liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PackageServerMojo.java

```diff
@@ -138,73 +183,82 @@ private void validateInclude() throws MojoFailureException {
             includeValues = new ArrayList<String>(includeStrings);
             for (int i = 0; i < includeValues.size(); i++) {
                 String value = includeValues.get(i);
-                includeValues.set(i, value.trim());
+                if (value.trim().length() > 0) {
+                    includeValues.set(i, value.trim());
+                }
             }
         } else {
             includeValues = new ArrayList<String>();
         }
+        return includeValues;
+    }
 
-        // if jar, validate include options, and add runnable
-        if (packageType.equals("jar")) {
-            if (includeValues.contains("usr") || includeValues.contains("wlp")) {
-                throw new MojoFailureException("Package type jar cannot be used with `usr` or `wlp`.");
-            }
+    /**
+     * Sets `packageFileType` based on include and packageType values. Validates the values specified are compatible.
+     * 
+     * @throws MojoFailureException
+     */
+    private void validateIncludeAndPackageType() throws MojoFailureException {
+        ArrayList<String> includeValues = parseInclude();
 
-            if (!includeValues.contains("runnable")) {
-                includeValues.add("runnable");
+        if (includeValues.size() > 1) {
+            if (includeValues.contains("runnable")) {
+                if (includeValues.contains("wlp") || includeValues.contains("usr")) {
+                    throw new MojoFailureException("The `include` parameter value `runnable` is not valid with `usr` or `wlp`.");
+                }
+            } else {
+                throw new MojoFailureException("The `include` parameter value `" + include + "` is not valid. The `include` parameter can be used with values `all`, `usr`, `minify`, `wlp`, `runnable`, `all,runnable`, and `minify,runnable`.");
```

Comment:
Seems like if we are going to do this validation, we should validate all the possible values, not just `runnable` in combination with `wlp` and `usr`.

Reply:
So you want me to make sure all values are valid? It wasn't doing that before.

Reply:
Yeah, but looking at it now it just stands out to me why we are only validating this scenario? I'm not sure what validation is already provided at the Liberty server command level, but we could probably also move this validation logic into ant since it would be consistent between Maven and Gradle.

Reply:
true...the command will catch the problem...should I open an issue to handle it in ci.ant?

Reply:
Removed validation of include values. The liberty command reports the error.